README.md

```diff
@@ -5,5 +5,99 @@ Mage-tools is an opinionated set of [mage](https://github.com/magefile/mage) tar
 [![Release](https://github.com/einride/mage-tools/actions/workflows/release.yml/badge.svg)](https://github.com/einride/mage-tools/actions/workflows/release.yml)
 
 ## Requirements
--	[Go](https://golang.org/doc/install) >= 1.17
--	[GNU Make](https://www.gnu.org/software/make/)
+
+- [Go](https://golang.org/doc/install) >= 1.17
+- [GNU Make](https://www.gnu.org/software/make/)
+
+## Getting started
+
+To initilize mage-tools in a repository, just run:
+
+```bash
+go run -mod=mod go.einride.tech/mage-tools -init
```

Comment:
IMO it's a bit surprising that `-init` is a flag and not a subcommand. Thinking of e.g. `terraform init`

Reply:
Hmm, true

Reply:
For initialization purposes I think it's better to specify a version than `-mod=mod`. I.e.:

`go run go.einride.tech/mage-tools@latest -init`

Reply:
Fixed